### ISSUE TITLE
nexe errors when run

### ISSUE THREAD

**Issue by ptmoy2** [jaredallard/nexe#163]
I can't get my application to work with nexe, so tried 'express-test' that was installed with nexe.  I copied the 'express-test' folder into my 'home' directory and ran nexe.  I got the following error:

```
$ cd express-test
$ nexe -i ./index.js
....> ERROR: option nodeConfigureArgs was empty
```

If I 'cd' into my project's directory and run nexe, I get this:

```
pm@T110:~/Projects/express/twTest$ ls
app.js  node_modules  package.json  public
pm@T110:~/Projects/express/twTest$ nexe -i ./app.js -o/ttt -v 4.2.2
1.0.3
```

nexe just comes back with '1.0.3', but no file is generated.  Running on Ubuntu 14.04

**Comment by lionel590** [comment]
Hi, same here

nexe -i webServer.js -o webServer.bin -r 4.2.1
....> ERROR: option nodeConfigureArgs was empty

I get the same error with
var nexe = require('/usr/local/lib/node_modules/nexe');

nexe.compile({
input: 'webServer.js',
output: '/home/pi/myProject/bin',
nodeVersion: '4.2.1',
nodeTempDir: '/home/pi/myProject/tmp',
python: '/usr/lib/python2.7',

flags: true,
framework: "nodejs"
}, function(err) {
console.log(err);
});

**Comment by ftwang** [comment]
if you add the following key to the compile options, it seems to work:
nexe.compile({
...
nodeConfigureArgs: [],
nodeMakeArgs: [] 
}, ...

**Comment by jaredallard** [comment]
Yes. Its a mistake of making them required


On Sun, Jan 24, 2016 at 2:01 AM, ftwang < notifications@github.com [notifications@github.com] > wrote:
if you add the following key to the compile options, it seems to work:
nexe.compile({
...
nodeConfigureArgs: [],
nodeMakeArgs: []
}, ...

—
Reply to this email directly or view it on GitHub
[https://github.com/jaredallard/nexe/issues/163#issuecomment-174274354] .[https://github.com/notifications/beacon/ACR9NcUCg4kb0wP6yShM7wf6tqe0dC3Kks5pdJiBgaJpZM4HKywG.gif]

**Comment by jaredallard** [comment]
Fixed in 1.0.4 :)
